core/designsystem/src/main/java/com/jerryalberto/mmas/core/designsystem/component/RoundedButton.kt

```diff
@@ -0,0 +1,76 @@
+package com.jerryalberto.mmas.core.designsystem.component
+
+import androidx.compose.foundation.BorderStroke
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.DateRange
+import androidx.compose.material3.Button
+import androidx.compose.material3.ButtonColors
+import androidx.compose.material3.ButtonDefaults
+import androidx.compose.material3.Icon
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.text.font.FontStyle
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.TextUnit
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.jerryalberto.mmas.core.designsystem.theme.Orange40
+
+@Composable
+fun RoundedButton(
+    shape: RoundedCornerShape,
+    border: BorderStroke,
+    buttonColors: ButtonColors,
+    iconLeft: ImageVector,
+    title: String,
+    fontWeight: FontWeight,
+    fontStyle: FontStyle,
+    fontSize: TextUnit,
+    padding: Dp,
+    onClick: () -> Unit = {}
+) {
+
+    Button(
+        onClick = onClick, shape = shape, border = border, colors = buttonColors
```

Comment:
we can implement the shape with https://m3.material.io/styles/shape/overview

Reply:
more information
![image](https://github.com/jchodev/MMAS-JC/assets/100594737/6310d017-8f5b-49f5-b312-07ae5c95e51f)